examples/with-solana/package.json

```diff
@@ -6,6 +6,7 @@
     "start": "pnpm tsx src/index.ts",
     "advanced": "pnpm tsx src/advanced.ts",
     "token-transfer": "pnpm tsx src/tokenTransfer.ts",
+    "allow-token-transfer": "pnpm tsx src/allowTokenTransferPolicy.ts",
```

Comment:
maybe a more generic/overarching name for this could be `token-transfer-with-policy`